### DISCUSSION TITLE
Is there an OAuth (or alike) implementation in this API?

### DISCUSSION THREAD

**Discussion by miguelgdx** [myConsciousness/atproto.dart#714] [Q&A] [answered]
I'm pretty much new to this API and the protocols used, but I'm wondering if, without requesting the user to give up his password, there is a way to have an user authorize or let a remote server/service access information from their profile or so.

**Comment by myConsciousness** [comment, marked as answer]
Hi @miguelgdx 

AT Protocol and Bluesky do not support OAuth now, so the user's Handle or Email and password are required for authentication.

But once OAuth is officially supported I will support it in my package :)
